### ISSUE TITLE
Problem with multiprocessing?

### ISSUE THREAD

**Issue by AjayTalati** [ikostrikov/pytorch-a3c#2]
Hi, thanks a lot for releasing this project. 

I wonder if you could give me some advice? I'm trying to adapt your code to a different environment. It's working great for a single thread, when I simple execute `train.py`,, without using torch multiprocessing. When I try to use it with multiprocessing, it still runs, but I get a screen full of error messages, like these,

```*** Error in `*** Error in `python': double free or corruption (out): 0x00007f6598000c80 ***
*** Error in `*** Error in `python': double free or corruption (out): 0x00007f6598000c80 ***
*** Error in `*** Error in `python': double free or corruption (out): 0x00007f6598000c80 ***```

I'm not sure whether it's learning either? I have a TensorFlow implementationn, so I know both my environment and A3C algorithm work.
 
I just wondered if you've seen this before? It might be something to do with the the `action` variable, because when I set it to a constant, the error goes away, but of course it doesn't learn then?

Thanks a lot for your help :)

**Comment by ikostrikov** [comment]
Hi, I need to see your code to understand what might go wrong.

**Comment by AjayTalati** [comment]
OK, thanks a lot for your offer of help. :)

It's part of a larger project, so I need to make it stand alone. I don't think the problems with your code as it works fine on a single processor. Also my code works fine asynchronously with TF? 

Will get back to you very soon! Cheers

**Comment by AjayTalati** [comment]
Hi, sorry about this - it seems this error was due to a combination of, [re-installing TF](https://github.com/tensorflow/tensorflow/issues/6968#issuecomment-279160301),
and my screwing up testing my project when porting things over from TensorFlow to PyTorch. 

I'm working with a Black Box simulator, and God (and probably the Devil too), only know how it works. Testing with it per simulation is costly time wise, and I've got an austere computational budget? 

I might have made a mistake testing, but as far as I can tell your implementation behaves the same as my other TF A3C implementation. So basically I think your codes great :+1: 

Thanks once again for releasing it :+1: 

On another note - I wonder if you have tried to implement your meta-optimiser for a single thread Actor Critic RL learner? Just curious, because I'm trying to do that now? Hopefully once the meta-optimiser is trained with a single learner, it can then be used for each of the learners in the full A3C?
